src/pages/gapsPatterns/GapsPatternsPage.tsx

```diff
@@ -89,16 +94,16 @@ function GapsByHour({ lineRef, operatorRef, fromDate, toDate }: BusLineStatistic
 
             <ComposedChart
               layout="vertical"
-              width={500}
+              width={widgetWidth}
```

Comment:
calculating the width is not needed - use responsive container
example:
https://recharts.org/en-US/examples/LineBarAreaComposedChart

Reply:
Thanks for the comment. I made a new commit.